schema/appdal/trigger.schema.xml

```diff
@@ -278,6 +283,8 @@
 
  <class name="TriggerDataHandler">
   <superclass name="ReadoutModule"/>
+  <relationship name="enabled_source_ids" description="List of source IDs enabled in this session: used by trigger to define the readout window for trigger decisions." class-type="SourceIDConf" low-cc="zero" high-cc="many" is-composite="no" is-exclusive="no" is-dependent="no"/>
+  <relationship name="mandatory_source_ids" description="Source Ids that will always be included in a trigger decision." class-type="SourceIDConf" low-cc="zero" high-cc="many" is-composite="no" is-exclusive="no" is-dependent="no"/>
```

Comment:
Why both `"TCDataProcessor"` and `"TriggerDataHandler"` (and therefore all the TAMakers, TCMakers too) need this?